core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1986,6 +1986,9 @@ private[spark] class DAGScheduler(
       logInfo("Host added was in lost list earlier: " + host)
       failedEpoch -= execId
     }
+    if (fileLostEpoch.contains(execId)) {
```

Comment:
This `if` is not needed as  removing a non-existing key from a `HashMap` gives back the `HashMap`(a few lines above it must be used because of the `logInfo`).

Reply:
I see. Ok, I'll remove this.